### ISSUE TITLE
Text Autocomplete no longer works

### ISSUE THREAD

**Issue by paul1149** [geany/geany#1595]
I was really enjoying geany's autocomplete function (for words. I use geany to edit text, not code), particularly the way it deletes the remaining part of an existing word you're overwriting (if that option is chosen).

Suddenly autocomplete no longer works at all. The only thing I can think is different is I changed the line height of the display, a la https://github.com/geany/geany/issues/1592, but I can't see how that would affect this. And I customized some theme colors.

I've tried jogging various settings, to no avail. I'm in a large document with a large word database, so that's not a problem.

Here's a shot of my settings. I don't see anything wrong with them. I'm on 1.30.1 on Linux. Thanks.

![preferences_004](https://user-images.githubusercontent.com/13952246/29945741-744647fc-8e70-11e7-986b-257d48e3fbf2.jpg)

**Comment by codebrainz** [comment]
You could try to run with a clean/fresh config like this:

```bash
$ geany -c /tmp/deleteme-later -v
```

Testing with a file that used to give your auto-completions.

You could also check (and/or paste here) the messages shown in `Help->Debug Messages`.

**Comment by paul1149** [comment]
Thanks for the idea. I did this by renaming my .config/geany folder, then running geany via shortcut. It created a fresh profile, and once I set autocomplete, there was no problem.

I then experimented in the original profile with a fresh filetypes.common file, but it made no difference. Then I removed the geany.conf file, forcing a new one, and the problem was solved.

So I think the geany.conf was corrupted somehow. I'll keep the new one and redo my settings. That should solve the problem I think. If so I'll close this thread.

Thanks again!
